src/schematools/permissions/auth.py

```diff
@@ -323,6 +312,12 @@ def _may_include_profile_table(self, profile_table: ProfileTableSchema):
             _match_filter_rule(rule, self._query_param_names) for rule in mandatory_filtersets
         )
 
+    def __str__(self) -> str:
+        """str implementation, for debugging purposes."""
+        self._materialize_profiles()
```

Comment:
deze call is hier toch niet nodig?

Reply:
Oh, ik had profiles en scopes door elkaar gehaald. Brainfart, ik haal het weg.